### PR TITLE
Add a new compose type for Hyper-V

### PR DESCRIPTION
**Pull request by dashea**
This is based on the VHD compose type, with the following differences:

  * Use the vhdx format instead of vhd
  * No WALinuxAgent
  * Install hyperv-daemons

The hyperv-daemons are activated through udev rules, so there is no need
to add them to the services line.

--- Description of proposed changes ---

--- Merge policy ---

- [ ] Travis CI PASS
- [ ] `*-aws-runtest` PASS
- [ ] `*-azure-runtest` PASS
- [ ] `*-images-runtest` PASS
- [ ] `*-openstack-runtest` PASS
- [ ] `*-vmware-runtest` PASS
- [ ] For `rhel8-*` and `rhel7-*` branches commit log references an approved
  bug in Bugzilla. Do not merge if the bug doesn't have the 3 ACKs set to `+`!

--- Jenkins commands ---

- `ok to test` to accept this pull request for testing
- `test this please` for a one time test run
- `retest this please` to start a new build

### CODE REVIEW

File: src/pylorax/api/compose.py [hunk 1]
```diff
@@ -848,6 +848,28 @@ def compose_args(compose_type):
                                  "app_template":            None,
                                  "app_file":                None,
                                 },
+            "hyper-v":          {"make_iso":                False,
```
Comment (bcl) [inline src/pylorax/api/compose.py]: This section needs to have all the new fields we've added, like tar_disk_name, etc.

Comment (dashea) [reply src/pylorax/api/compose.py]: oops. Fixed.

File: src/pylorax/api/compose.py [hunk 2]
```diff
@@ -848,6 +848,31 @@ def compose_args(compose_type):
...
+                                 "make_disk":               True,
+                                 "make_fsimage":            False,
+                                 "make_appliance":          False,
+                                 "make_ami":                False,
+                                 "make_tar":                False,
+                                 "make_tar_disk":           True,
```
Comment (bcl) [inline src/pylorax/api/compose.py]: Should be false :)

Comment (dashea) [reply src/pylorax/api/compose.py]: wow, was I braindead yesterday or what. Fixed.

**Pull request merged**